azure-communication-ui/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
     }
 
     ext {
-        call_library_version_name = '1.11.0'
+        call_library_version_name = '1.13.0-beta.1'
```

Comment:
do we want to bump this in develop, or against a beta release branch?

Reply:
It will be updated in release branch to correct one